pontoon/base/migrations/0001_squashed_0154_auto_20200206_1736.py

```diff
@@ -3378,320 +842,76 @@ class Migration(migrations.Migration):
                 (
                     "performed_by",
                     models.ForeignKey(
-                        on_delete=django.db.models.deletion.CASCADE,
+                        null=True,
+                        on_delete=django.db.models.deletion.SET_NULL,
                         related_name="changed_permissions_log",
                         to=settings.AUTH_USER_MODEL,
                     ),
                 ),
                 (
                     "performed_on",
                     models.ForeignKey(
-                        on_delete=django.db.models.deletion.CASCADE,
+                        null=True,
```

Comment:
This will require a few additional changes e.g. [here](
https://github.com/mozilla/pontoon/blob/d73095890882b65e6c7d5c73da1546ae8bdf130c/pontoon/base/admin.py#L355).

Reply:
I don't think so. We're not actually making changes to the model, we're just squashing the schema part of migrations 0001 and 0002 into 0001, and the data part of migrations 0001 and 0002 into 0002.

See:
https://github.com/mozilla/pontoon/pull/1881/files#diff-eb0ed65c4398ca63ffc99e9d41a7417e1539dd0976ce378ad8219647e7f29b56L38

Reply:
Ah, I've got confused by the way how the diff was displayed.